Formula/graph-tool.rb

```diff
@@ -3,8 +3,8 @@ class GraphTool < Formula
 
   desc "Efficient network analysis for Python 3"
   homepage "https://graph-tool.skewed.de/"
-  url "https://downloads.skewed.de/graph-tool/graph-tool-2.27.tar.bz2"
-  sha256 "4740c69720dfbebf8fb3e77057b3e6a257ccf0432cdaf7345f873247390e4313"
+  url "https://downloads.skewed.de/graph-tool/graph-tool-2.28.tar.bz2"
+  sha256 "a2dbe0c926c058b1a6f696a05b9d9b65a3ceed57b8b6b0fe1474076bca9f8111"
   revision 6
```

Comment:
I think this can be removed which cause build failure:
```
12:13:09 ==> brew audit graph-tool --online
12:14:16 ==> FAILED
12:14:16 ==> Installing 'bundler' gem
12:14:16 Successfully installed bundler-2.0.1
12:14:16 1 gem installed
12:14:16 Error: 1 problem in 1 formula detected
12:14:16 graph-tool:
12:14:16   * 'revision 6' should be removed
```

Reply:
Done.